content/_code-samples/secure-signing/py/sign-payment.py

```diff
@@ -0,0 +1,34 @@
+from xrpl.core import keypairs 
+
+# Define network client
+from xrpl.clients import JsonRpcClient
+JSON_RPC_URL = "https://s.altnet.rippletest.net:51234/"
+client = JsonRpcClient(JSON_RPC_URL)
```

Comment:
If you notice, the ripple-lib example doesn't connect to the network. That's intentional, since the most secure signing takes place on a machine that's not online at all. The equivalent is possible with xrpl-py.